src/components/Modal/Modal.jsx

```diff
@@ -18,32 +21,27 @@ const preRender = (
   restProps,
   size,
 ) => (
-  <div
-    className={styles.backdrop}
+  <dialog
+    {...transferProps(restProps)}
+    className={classNames(
+      styles.root,
+      getSizeClassName(size, styles),
+      getPositionClassName(position, styles),
+    )}
     onClick={(e) => {
+      e.stopPropagation();
```

Comment:
The effect of this is way too big, don't you think…? Why do we need it?

Reply:
This is part of removing functionality commented in https://github.com/react-ui-org/react-ui/pull/544#discussion_r1643152681

Reply:
Removed.